lib/config/global.ts

```diff
@@ -67,7 +69,26 @@ export class GlobalConfig {
     return result;
   }
 
+  static getExperimentalFlag(key: string): boolean {
+    const experimentalFlags = GlobalConfig.get('experimentalFlags');
+
+    if (!experimentalFlags) {
```

Comment:
```suggestion
    if (!experimentalFlags?.length) {
```

early return if emtpy